### PR TITLE
Group 9 - Case Study #1

### PR DESCRIPTION
**Pull request by Amanpreet1304**

### CODE REVIEW

**Comment by 0matleb2** [comment]
I would have suggested waiting for at least one code review before merging.

File: robocode.battle/src/main/java/net/sf/robocode/battle/Module.java [hunk 1]
```diff
@@ -19,6 +19,8 @@
  * @author Pavel Savara (original)
  */
 public class Module {
+
+
```
Comment (0matleb2) [inline robocode.battle/src/main/java/net/sf/robocode/battle/Module.java:23]: Unnecessary whitespace changes.

File: robocode.battle/src/main/java/net/sf/robocode/recording/RecordManager.java [hunk 2]
```diff
@@ -110,15 +110,21 @@ public void detachRecorder() {
         recorder.detachRecorder();
     }
 
+    private void deleteTempFile(){
```
Comment (0matleb2) [inline robocode.battle/src/main/java/net/sf/robocode/recording/RecordManager.java:113]: The code formatting in this block doesn't match the project conventions.

**Comment by rkerw** [comment]
@0matleb2 Oh sorry! We didn't know this was going to be formally reviewed, and we merged according to the handout for C1. Should we proceed to fix these issues, or keep them in mind for the next deliverable? Sorry for the inconvenience.

File: robocode.battle/src/main/java/net/sf/robocode/battle/Module.java (hunk 1 above)
Comment (Amanpreet1304) [reply robocode.battle/src/main/java/net/sf/robocode/battle/Module.java:23]: Removed whitespace.

File: robocode.battle/src/main/java/net/sf/robocode/recording/RecordManager.java (hunk 2 above)
Comment (Amanpreet1304) [reply robocode.battle/src/main/java/net/sf/robocode/recording/RecordManager.java:113]: Code Formatting corrected.
